plugins/airplay/Dockerfile.template

```
@@ -8,9 +8,9 @@ SHELL ["/bin/sh", "-eo", "pipefail", "-c"]
 
 # shairport-sync docker image doesn't include pulseaudio support so we use ALSA bridge
 ENV PULSE_SERVER=tcp:localhost:4317
-RUN apk update && apk add --no-cache supervisor curl~=7 && \
-  curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh \
-  && apk del curl
+RUN apk upgrade --no-cache && apk add --no-cache supervisor curl
+RUN curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh
+RUN apk del curl
```

```suggestion
RUN apk upgrade --no-cache && apk add --no-cache supervisor curl && \
    curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh \
    apk del curl
```

Any reason why these steps can't happen together?

---

I am not sure of the exact reason, but I had issues with building the `airplay` container without splitting these steps. 
It was mostly trial and error...

--- 


I tried again with your suggestion and got the error below 
```
Error: The command '/bin/sh -eo pipefail -c apk upgrade --no-cache && apk add --no-cache supervisor curl &&     curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh     apk del curl' returned a non-zero code: 85
```

--- 

I tried again with 
```
RUN apk upgrade --no-cache
RUN apk add --no-cache supervisor curl && \
    curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh \
    apk del curl
```

And got the error below 
```
[Error]             Some services failed to build:
[Error]               Service: airplay
[Error]                 Error: The command '/bin/sh -eo pipefail -c apk add --no-cache supervisor curl &&     curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh     apk del curl' returned a non-zero code: 2
```

--- 

Finally...I tried this 
```
I am not sure of the exact reason, but I had issues with building the `airplay` container without splitting these steps. 
It was mostly trial and error...

--- 


I tried again with your suggestion and got the error below 
```
Error: The command '/bin/sh -eo pipefail -c apk upgrade --no-cache && apk add --no-cache supervisor curl &&     curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh     apk del curl' returned a non-zero code: 85
```

--- 

I tried again with 
```
RUN apk upgrade --no-cache
RUN apk add --no-cache supervisor curl && \
    curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh \
    apk del curl
```

And got the error below 
```
[Error]             Some services failed to build:
[Error]               Service: airplay
[Error]                 Error: The command '/bin/sh -eo pipefail -c apk add --no-cache supervisor curl &&     curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh     apk del curl' returned a non-zero code: 2
```

--- 

Finally I tried with 
```
RUN apk upgrade --no-cache && apk add --no-cache supervisor curl && \
    curl -skL https://raw.githubusercontent.com/balena-io-experimental/audio/master/scripts/alsa-bridge/alpine-setup.sh | sh
RUN apk del curl
```
  
And this worked...

---

I just updated the branch